src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import type { ReportHandler } from 'web-vitals';
```

Comment:
Just curious what is this file for? 💭

Reply:
This is a file generated from CRA. Seems like it is to send web metrics. 

https://create-react-app.dev/docs/measuring-performance/#sending-results-to-analytics

We can pass the ReportHandler, a function to send the metrics out. Right now we don't pass in any ReportHandler, So it is not actually being used. But the easiest way is we can pass `console.log`, and it will log the metrics out.

For example on `index.tsx`, we put `reportWebVitals(console.log);`, the metrics are logged on the console like this:

![Screen Shot 2565-08-05 at 15 13 27](https://user-images.githubusercontent.com/6965195/183033731-e1c7d165-bc87-4b50-8eab-a2ceb354aae7.png)

Right now, by default, it logs the metrics below which are CLS, FID, FCP, LCP and TTFB

But I also don't know what each metrics are for. Never play with it before 😅